www/header.php

```diff
@@ -204,3 +242,13 @@
     <div id="cv-playlist">
         <ul class="cv-playlist"></ul>
     </div>
+
+	<!-- Only included when generate index.html for developmed purpose -->
+	<!--=include templates/indextpl.html -->
+	<!--=include footer.php -->
+
+<!-- make wellformed html; correct unclosed body and html (normally done by footer ) -->
+<!-- GEN_DEV_INDEX_TAG
```

Comment:
To make html precocessing etc possible like the bundle generation, the html should be wellformatted